### DISCUSSION TITLE
Characters for building new symbols

### DISCUSSION THREAD

**Discussion by markjago** [KaTeX/KaTeX#3685] [Q&A]
Hi everyone, I'm defining new logic symbols (e.g. left-facing versions of \models, \vdash, \Vdash etc). They come out very big when built from '|' '-' etc, compared to the primitive \Vdash, but too small using \shortmid etc. I tried scaling with CSS, which is not only a pain but also gives too-thin line widths. Are there primitive characters accessible in Katex (e.g via \char"XXX) which are shorter than '|' but longer than \shortmid? Thanks!

**Comment by edemaine** [comment]
You can load the fonts via https://opentype.js.org/ and see what symbols are available. If `\shortmid` is shorter than you need, perhaps you could get what you want by overlapping and stacking multiple `\shortmid`s?
